src/shared_utilities/drivers.jl

```diff
@@ -625,7 +678,7 @@ This requires the field `ρ_sfc` to be present in the cache `p` under the name
 of the model.
 """
 function surface_air_density(
-    atmos::CoupledAtmosphere,
+    atmos::Union{CoupledAtmosphere, NamedTuple},
```

Comment:
I changed this because when we compute turbulent fluxes from the coupler, we pass a NamedTuple as "atmos" here